pkg/volume/volume.go

```diff
@@ -177,8 +177,8 @@ type Attacher interface {
 
 	// VolumesAreAttached checks whether the list of volumes still attached to the specified
 	// the node. It returns a map which maps from the volume spec to the checking result.
-	// If an error is occured during checking, the error will be returned
-	VolumesAreAttached(specs []*Spec, nodeName types.NodeName) (map[*Spec]bool, error)
+	// If an error is occurred during checking, the error will be returned
+	VolumesAreAttached(volumesByNode map[types.NodeName][]*Spec) (map[*Spec]bool, error)
```

Comment:
AFAIK, GCE can attach one read-only volume to multiple nodes. With return value of type `map[*Spec]bool` you loose information which node actually misses the volume.

https://cloud.google.com/compute/docs/disks/add-persistent-disk#use_multi_instances

@thockin, @saad-ali, do we want to support this in Kubernetes?